dist/es5src/drivers/indexeddb.js

```diff
@@ -0,0 +1,576 @@
+(function (global, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define('asyncStorage', ['exports', 'module'], factory);
+    } else if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
+        factory(exports, module);
+    } else {
+        var mod = {
+            exports: {}
+        };
+        factory(mod.exports, mod);
+        global.asyncStorage = mod.exports;
```

Comment:
I think that it should be a separate PR. This one is already large enough to make tracking changes difficult.
Would be happy to open a new PR if you have suggested names on mind.
We just need to change:
* the driver names exposed by the localforage object
* the name property on each driver and
* the module mapping function inside grunt.js that babel uses